renderer/src/main.tsx

```diff
@@ -14,3 +14,12 @@ ReactDOM.createRoot(
     </BrowserRouter>
   </React.StrictMode>
 )
+
+window.electron.getActivityLog().then(log => {
+  console.log('GOT INITIAL ACTIVITY LOG')
+  log.forEach(entry => console.log('[ACTIVITY] %j', entry))
+})
+
+window.electron.onActivityLogged(entry => {
+  console.log('[ACTIVITY] %j', entry)
+})
```

Comment:
This is a temporary UI to verify that the new infrastructure for ActivityLog works as intended.

Later, we can replace this either with some crude UI, or wait for the proper UI implementation from Subvisual.

I'd like to defer any UI work after this PR is landed.